modules/config/pages/eap-config.adoc

```diff
@@ -1,11 +1,159 @@
 = JBoss EAP Configuration
 
-Brief blurb about the different ways to configure JBoss EAP....
+== Overview of Standalone Server
 
-== Section Header
+* Standalone servers are perfect for running a single instance of Red Hat JBoss EAP.
+* All configurations are managed in the `JBOSS_HOME/standalone/configuration/standalone.xml` file.
```

Comment:
not true. standalone can also run standalone-ha, standalone-full and full-ha.xml